static/css/style.css

```diff
@@ -1170,6 +1174,10 @@ div#container-display-show {
     #key-padding {
         padding-bottom: 4px;
     }
+
+    #showRoot {
+        width: 49%;
+    }
```

Comment:
when a root is selected, if width is 49% it will only show half of it and user won't be able to distinguish which root is selected